src/com/jester/controller/Controller.java

```diff
@@ -65,6 +66,13 @@ public String[] GetAllStoredAlbums() {
        return albums;
     }
     
+    public String GetLyricsPath(String fetch_lyrics) {
+        String lyrics_path;  
+        lyrics_path = handle.getLyricContentPath(fetch_lyrics);
+        
+        return lyrics_path;

```

Comment:
String return of lyrics_path  which is okay because we need to get the file DIR of the .txt lyrics file